web/src/left_sidebar_navigation_area_popovers.js

```diff
@@ -41,6 +41,19 @@ function register_mark_all_read_handler(event) {
     instance.hide();
 }
 
+function register_toggle_display_left_sidebar_messsage_count_handler(event) {
+    const {instance} = event.data;
+    const data = {};
+    const home_view_unread_msg_count = user_settings.web_left_sidebar_unreads_count_summary;
+    data.web_left_sidebar_unreads_count_summary = JSON.stringify(!home_view_unread_msg_count);
```

Comment:
This is a weird way to construct `data`; and worse than the similar blocks just above; can you follow those better conventions?

Reply:
Thank you for your review. I appreciate your feedback. Regarding the construction of `data` I took reference from a similar block below:

```
$popper.one("click", "#toggle_display_starred_msg_count", () => {
                const data = {};
                const starred_msg_counts = user_settings.starred_message_counts;
                data.starred_message_counts = JSON.stringify(!starred_msg_counts);

                channel.patch({
                    url: "/json/settings",
                    data,
                });
                instance.hide();
            });
```

Reply:
Hmm, maybe the right plan is to refactor those as well; the ones above this seem more sensible.